server/src/test/java/org/elasticsearch/action/admin/indices/resolve/ResolveClusterActionResponseTests.java

```diff
@@ -52,6 +63,6 @@ protected Writeable.Reader<ResolveClusterActionResponse> instanceReader() {
 
     @Override
     protected ResolveClusterActionResponse mutateInstance(ResolveClusterActionResponse response) {
-        return new ResolveClusterActionResponse(randomResolveClusterInfoMap());
+        return new ResolveClusterActionResponse(randomResolveClusterInfoMap(() -> randomResolveClusterInfoWithErrorString()));
```

Comment:
This is fine, but alternatively you could use the "randomValueOtherThan" helper method [from ESTestCase](https://github.com/elastic/elasticsearch/blob/main/test/framework/src/main/java/org/elasticsearch/test/ESTestCase.java#L1208) that randomizes until it has created an object thats not equal to the input. Might save a bit of extra code here and be a bit more readably on the long run.

Reply:
Oh, cool. Thanks for the tip. Very helpful method! I've changed the code to use on the next push.